app/scripts/components/common/cookie-consent/index.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { useState, useEffect } from 'react';
 import { Icon } from '@trussworks/react-uswds';
-import { COOKIE_CONSENT_KEY } from './utils';
+import { useLocation } from 'react-router-dom';
```

Comment:
react-router-dom will be a trouble when exported as a library (and used in Next instance.) :[  What will be the best way to replace this logic? -